### ISSUE TITLE
<All nations> - <Crash after a month or two of gameplay>

### ISSUE THREAD

**Issue by Vincentius267** [Darkest-Hour-Hoi4/Darkest-Hour-Issues#139]
### OS

Windows 11

### Any Expansions?

Nothing

### Are you using Steam?

Yes i am

### Were you playing in multiplayer?

Yes i was

### Hoi4 Version

12.12.14 

### DH Version (Current one is 1.0.4-hotfix1)

Steam version (1.0)

### Other Mods

None

### Description

```bash
Irrespective of what nation I'm playing as, the game crashes after a month or two of gameplay.
```

### Reproduction steps

```bash
1. Go to '...'
2. Click on '...'
3. Select '...'
?
```

### Screenshots/Video

N/A

### Logs

N/A

### Savegame

[Italy DH.zip](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11843931/Italy.DH.zip)

**Comment by Darijus11** [comment]
Can you send me an error log. 
They can be found under Documents\Paradox Interactive\Hearts of Iron IV\crashes

**Comment by Vincentius267** [comment]
[crashlog.zip](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11850849/crashlog.zip)

**Comment by Darijus11** [comment]
We've already identified the issue and fixed it. We will deploy the fix with 1.0.5 version. The 1.0.5 version will be deployed when ready. In the meantime, keep 3D units off in the settings.
